test/lib.bash

```diff
@@ -50,22 +50,17 @@ function run_e2e_tests {
     logger.error 'Tests have failures!'
   fi
 
-  wait_for_knative_serving_ingress_ns_deleted || return 1
```

Comment:
Why are you removing this? Is this intentional?

Reply:
Yeah, because the namespace should already be removed if the test runs fine.

Reply:
Actually, going to adjust the test a bit to make sure it's properly gone.

Reply:
Or just add this line back 😂

Reply:
I added this as a workaround for Azure. There's a bug which requires patching the kourier ingress. Not sure if the Golang test ensures that.

Reply:
OK. You added it back:) Thanks